examples/verify-common.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash -e
+
+DELAY="${DELAY:-0}"
+DOCKER_NO_PULL="${DOCKER_NO_PULL:-}"
+MANUAL="${MANUAL:-}"
+NAME="${NAME:-}"
+PATHS="${PATHS:-.}"
+UPARGS="${UPARGS:-}"
+
+
+run_log () {
+    echo -e "\n> [${NAME}] ${*}"
+}
+
+bring_up_example_stack () {
+    local args path up_args
+    args=("${UPARGS[@]}")
+    path="$1"
+    if [[ -z "$DOCKER_NO_PULL" ]]; then
+	run_log "Pull the images ($path)"
+	read -ra up_args <<< "up --build -d ${args[*]}"
+    else
+	# this prevents docker pulling the load images in ci
+	docker-compose build || return 1
```

Comment:
i think this is unneccessary - the problem was that the  images werent tagged - ill update...